cmd/ndt7-client/internal/summary.go

```diff
@@ -0,0 +1,93 @@
+package internal
+
+import (
+	"strings"
+
+	"github.com/m-lab/ndt7-client-go"
+	"github.com/m-lab/ndt7-client-go/spec"
+)
+
+// valueUnitPair represents a {"Value": ..., "Unit": ...} pair.
+type valueUnitPair struct {
+	Value float64
+	Unit  string
+}
+
+// Summary is a struct containing the values displayed to the user at
+// the end of an ndt7 test.
+type Summary struct {
+	// FQDN of the server used for this test.
+	Server string
+
+	// IP address of the client.
+	Client string
```

Comment:
Would you mind naming this variable `ClientIP`?

Reply:
At that point I would also need to rename `Server` to `ServerFQDN`, for the same reason.

I originally had `ServerFQDN` and `ClientIP` as field names there, which translated to `server` and `client` in the JSON and you advised me to avoid using the `json:"fieldname"` annotation but rather re-use the same field names in the source code.

Did you change your mind, or you're suggesting to have `{"ServerFQDN": ..., "ClientIP": ...}` in the summary JSON, too?

Reply:
When I mentioned to avoid using JSON annotations, I said this because we applied a similar change to the ndt-server in the past, so I wanted to help to enforce consistency in this respect. I don't like the fact that in a JSON variables have upper case names, but I understand how the annotation complicates maintainability.

I believe my comment regarding changing the name of the variable was caused by the documentation string, which starts with "IP", which led me to think in terms of the ClientIP, I guess.

Anyway, variable names can be changed at any time and hence it does not matter.